sources/nodes/external.go

```diff
@@ -27,16 +27,22 @@ import (
 
 // Provides a list of external cadvisor nodes to monitor.
 type externalCadvisorNodes struct {
+	// File containing list of hosts to monitor.
 	hostsFile string
-	nodes     *NodeList
+
+	// Whether to run in "standalone" mode and only consider localhost.
+	standalone bool
+
+	nodes *NodeList
 }
 
 // While updating this, also update heapster/deploy/Dockerfile.
 var hostsFile = flag.String("external_hosts_file", "/var/run/heapster/hosts", "A file that heapster refers to get a list of nodes to monitor.")
+var standaloneMode = flag.Bool("standalone", false, "Whether to run Heapster in \"standalone\" mode where it only targets the current node.")
```

Comment:
Today we can't target the local Kubelet since we get the pods from the apiserver no? When we change that we can move the flag to heapster.go and take it as a param here I think.